src/openrct2/world/scenery.h

```diff
@@ -67,7 +67,7 @@ typedef enum {
     SMALL_SCENERY_FLAG15 = (1 << 14),                           // 0x4000
     SMALL_SCENERY_FLAG16 = (1 << 15),                           // 0x8000
     SMALL_SCENERY_FLAG17 = (1 << 16),                           // 0x10000
-    SMALL_SCENERY_FLAG18 = (1 << 17),                           // 0x20000
+    SMALL_SCENERY_FLAG_STACKABLE = (1 << 17),                   // 0x20000
```

Comment:
Change comment to `// 0x20000; means scenery item can be placed in the air and over water